src/data/adventures.txt

```diff
@@ -230,8 +230,8 @@ Pirate	adventure=160	Env: indoor Stat: 140	Belowdecks
 IsleWar	adventure=134	Env: indoor Stat: 165	Wartime Frat House (Hippy Disguise)	1 beer helmet, 1 distressed denim pants, 1 bejeweled pledge pin
 IsleWar	adventure=135	Env: indoor Stat: 0	Wartime Frat House	1 beer helmet, 1 distressed denim pants, 1 bejeweled pledge pin|1 bullet-proof corduroys, 1 round purple sunglasses, 1 reinforced beaded headband
 
-IsleWar	adventure=131	Env: indoor Stat: 165	Wartime Hippy Camp (Frat Disguise)	1 bullet-proof corduroys, 1 round purple sunglasses, 1 reinforced beaded headband
-IsleWar	adventure=133	Env: indoor Stat: 0	Wartime Hippy Camp	1 bullet-proof corduroys, 1 round purple sunglasses, 1 reinforced beaded headband|1 beer helmet, 1 distressed denim pants, 1 bejeweled pledge pin
+IsleWar	adventure=131	Env: indoor Stat: 0	Wartime Hippy Camp	1 bullet-proof corduroys, 1 round purple sunglasses, 1 reinforced beaded headband|1 beer helmet, 1 distressed denim pants, 1 bejeweled pledge pin
+IsleWar	adventure=133	Env: indoor Stat: 165	Wartime Hippy Camp (Frat Disguise)	1 bullet-proof corduroys, 1 round purple sunglasses, 1 reinforced beaded headband
 
```

Comment:
The Wiki disagrees with you.

- In either frat disguise = 131
- In either hippy disguise or undisguised = 133

Which is the way it was.

https://kol.coldfront.net/thekolwiki/index.php/The_Hippy_Camp_(Verge_of_War)

The bug was in AdventurePool, which had the WARTIME_FRAT_HOUSE_DISGUISED and WARTIME_HIPPY_CAMP_DISGUISED constants swapped.